Gemfile

```diff
@@ -3,6 +3,7 @@ source "https://rubygems.org"
 gem "github-pages", group: :jekyll_plugins
 gem "yard"
 gem "yard-sorbet"
+gem "redcarpet"
```

Comment:
Could i trouble you to slot this alphabetically? (There should be a `Bundler/OrderedGems` violation, but i guess we don't have the wired up.)